packages/next/src/generators/library/library.ts

```diff
@@ -71,36 +71,6 @@ export async function libraryGenerator(host: Tree, rawOptions: Schema) {
   );
   addTsConfigPath(host, `${options.importPath}/server`, [serverEntryPath]);
 
-  updateJson(
```

Comment:
This never worked well, and the user should already have emotion and styled-components in the app. As per this PR (https://github.com/nrwl/nx/pull/17286) the app's babel config will now be applied to the libs as well. You never want to be in a situation where different styling solutions are used between apps and libs without a build step in the lib to normalize to CSS or something.

Reply:
We'll need a follow-up PR to enable emotion and styled-components in SWC for Next.js since they've added some early support for them.